### DISCUSSION TITLE
V2导出设置失败

### DISCUSSION THREAD

**Discussion by Kulahala** [the1812/Bilibili-Evolved#2846] [提问 / Q&A] [answered]
应该不是我网络问题吧
![Snipaste_2022-01-06_18-54-23](https://user-images.githubusercontent.com/54696830/148372566-4464b53e-6d6f-40ac-9a51-bf20295229c4.png)

**Comment by the1812** [comment, marked as answer]
是之前 spmid 的问题 (#2247), 这玩意现在提高了反破坏机制, 删掉相应数据后仍然能够运行并干扰下载链接

**Reply by the1812** [reply]
不过可以先去动态首页再导出
